### DISCUSSION TITLE
Is there a way to launch Explorer Patcher 'Properties' besides right-mousing on the taskbar?

### DISCUSSION THREAD

**Discussion by kwhopson** [valinet/ExplorerPatcher#1930] [Q&A]
Hi, 

I installed Explorer Patcher less than two weeks ago on three different Windows 11 PCs, and right-moused on the taskbar of all three to access Explorer Patcher Properties, thankful to be able to change the Start Menu default to All Apps. 

Shortly after, the ability to right-mouse on the taskbar of all three changed, now no longer able to see Taskbar Manager, Taskbar Settings or Explorer Patcher Properties. When I right-mouse on the taskbar, all icons disappear for a second, then return, but Manager, Settings or Properties for Explorer Patcher never show. 

Though I can no longer get to Explorer Patcher Properties, the tweaks to Windows 11 that I initially used it for, remain.  So I wonder if there is an alternate route for launching Explorer Patcher besides right-mousing on the taskbar?

Thanks much, 
kwhopson

**Comment by MightyM8** [comment]
I am also experiencing this inability to access Properties screen for EP (#1929). However, I am not experiencing the same as OP regarding prior changes made still holding.

**Reply by tmaleski** [reply]
I'm having the same issue using 22621.1992.56.1. The notification window is telling me there is an update, 22621.1992.56.3, but when I right-click on the task bar, it appears to perform the reload function.

**Reply by kwhopson** [reply]
Exactly, though the three anticipated options when right-mousing on the taskbar are never presented (Taskbar Manager, Taskbar Settings and most recently, Explorer Patcher Properties).  This is why I ask if there is an alternate way to launch Explorer Patcher, since it does not end up as an app in the Start Menu or have an otherwise launchable executable.

**Comment by pyrates999** [comment]
to run the explorer patcher properties application, run this from the run menu:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI

Then go to advanced and make sure the option **Do not change the taskbar context menu (e.g. do not display the "Properties"** is not enabled. Then you can right click on the taskbar and you should see the **Properties** option.

**Reply by tmaleski** [reply]
Using the above I was able to access the dialog. My setting was not enabled already. I turned on and then back off, still not working.

Windows 11 Pro v10.0.22621 Build 22621

![image](https://github.com/valinet/ExplorerPatcher/assets/14281057/61b1b242-4bc3-4805-aaa7-5e634dadbb99)

**Reply by tmaleski** [reply]
WINVER: Windows 11 Pro Version 22H2 (OS Build 22621.2134)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.

**Reply by tmaleski** [reply]
it was posted in the above post but here it is again, Windows 11 Pro v10.0.22621 Build 22621

**Reply by pyrates999** [reply]
You need to run winver and show the build number from there.

**Reply by tmaleski** [reply]
WINVER: Windows 11 Pro Version 22H2 (OS Build 22621.2134)

**Comment by MightyM8** [comment]
Windows 11 23H2 build 22631.2262
I am also experiencing the same as OP.
I was able to access the properties screen by using recommended method. I can confirm that the setting was NOT enabled.
Still no access to the Properties in the context menu on Taskbar ricghtclick; Also, none of the settings selected (Windows 10 style taskbar for example) are taking effect.

**Reply by pyrates999** [reply]
Insider builds are not supported by EP: https://blogs.windows.com/windows-insider/2023/08/24/announcing-windows-11-insider-preview-build-22621-2262-and-22631-2262-beta-channel/

**Comment by SouthernHeftruck** [comment]
Regedit: HKEY_CURRENT_USER\Software\ExplorerPatcher\OpenPropertiesAtNextStart  set value to 1 and maybe reboot worked for me to regain access to the properties.  Now I was able to uncheck  Do not change the taskbar context menu (e.g. do not display the "Properties"

**Comment by kwhopson** [comment]
OP here.  Thanks for all the dialogue. Still no solution. Here's my update.

By adding the DWORD as SouthernHeftruck suggested I was able to launch Explorer Patcher Properties again.

But I found, just like tmaleski and MIghtyM8, 'Do not change the taskbar context menu' was **already disabled**. 

As a test, I enabled it and the two default options, Taskbar Manager and Taskbar Setting, returned, without Explorer Patcher Properties of course.. 

Disabled again and it went back to no response at all from right-mousing except for an possibly an Explorer refresh.

As another test while I had Explorer Patcher Properties open, I **switched** the 'style' to Windows 10 (under Taskbar) and found that right-mousing on the taskbar then brings up a Properties option that will launch Explorer Patcher Properties.  Not Ideal, just an option, **maybe** a clue.

WIndows 11 Enterprise | Version 22H2 | OS Build 22621.2134

kwhopson
